### DISCUSSION TITLE
Jump of loss when resume training from checkpoint

### DISCUSSION THREAD

**Discussion by BZandi** [Lightning-AI/pytorch-lightning#7097] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I noticed that the loss jumps every time I resume the training (see figure). I use 'resume_from_checkpoint' to continue the training. The parameters seem to load correctly, including the optimisation state. Does anyone know where this behaviour comes from, or is it normal? Thanks in advance for any ideas

```Python
# Save model
trainer.save_checkpoint('Path')

# Load model and resume training
Model = FF_Class.FeedForwardShuffle.load_from_checkpoint('Path to Checkpoint')
trainer = pl.Trainer(resume_from_checkpoint=Max_Checkpoint,...)

# Start training
trainer.fit(Model)
```

![FF_QP_1_TargetCH_1_BatchSize_1000_LR_0 01](https://user-images.githubusercontent.com/62992112/115233004-a933dc00-a117-11eb-9ad0-0f261caae7cf.png)
